aws/resource_aws_ec2_traffic_mirror_target.go

```diff
@@ -0,0 +1,120 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceAwsEc2TrafficMirrorTarget() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsEc2TrafficMirrorTargetCreate,
+		Read:   resourceAwsEc2TrafficMirrorTargetRead,
+		Delete: resourceAwsEc2TrafficMirrorTargetDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"network_interface_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				ConflictsWith: []string{
+					"network_load_balancer_arn",
+				},
```

Comment:
If the API requires one or the other of these two attributes, then we can use `ExactlyOneOf` instead of `ConflictsWith` to have Terraform return an error if both are missing or both are declared. 👍 

```go
				ExactlyOneOf: []string{
					"network_interface_id",
					"network_load_balancer_arn",
				},
```